Raven.Client.Lightweight/Document/AbstractDocumentQuery.cs

```diff
@@ -603,8 +604,13 @@ protected internal QueryOperation InitializeQueryOperation()
 
         public IndexQuery GetIndexQuery(bool isAsync)
```

Comment:
This ```isAsync``` property is not used and is misleading. Removing it is simple so it might worth the effort.